### DISCUSSION TITLE
橡胶垫和数值垫有放置bug

### DISCUSSION THREAD

**Discussion by ShangguanXi** [IndustrialCraftMC/IndustrialCraft2#80] [【漏洞反馈】Bugs Report]
https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/78731793/f7d34544-7ba2-4206-b1c9-726d7ffd4aca

https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/78731793/3ac54879-b3ee-4e73-a5bd-0d9a25a61bd2

**Comment by AbbottMc** [comment]
如果你说的是浮空的话，它们就是允许浮空放置的

**Comment by AbbottMc** [comment]
已修复
